### DISCUSSION TITLE
SiameseKPConv without providing pre trained models

### DISCUSSION THREAD

**Discussion by pingcong** [community/community#124530] [Programming Help]
### Select Topic Area

Question

### Body

Hello, I am very interested in your proposed SiameseKPConv network for 3D change detection. The detection effect mentioned in the paper reaches state-of-the-art, and I want to reproduce it myself. The Git repository provides limited information, does it have pre trained weights? I hope to verify the effect on my own dataset. Thank you very much!

**Comment by LiteBrite82** [comment]
Hi @pingcong ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the [Torch Points3D](https://github.com/torch-points3d) repository. 

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
